### PR TITLE
[PAGOPA-753] selfcare repository

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- update codeowners file

#### Motivation and Context

- necessary in the creation of the repository

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # see https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file
 
-* @pagopa/pagopa-tech
+* @pagopa/selfcare-integration
```
Comment (jacopocarlini) [inline CODEOWNERS:3]:
```suggestion
* @pagopa/pagopa-team-core
```

Comment (alessio-acitelli) [reply CODEOWNERS]: done

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

File: docker/config/.env [hunk 2]
```diff
@@ -1,8 +1,7 @@
-# TODO: put docker env vars here
 ENV=docker
-# to use azurite
-#CONNECTION_STRING=UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://azure-storage
-
-# to use cosmosDB
-#COSMOS_URI=http://10.20.0.2:8081/
-#COSMOS_KEY=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==
+APP_LOGGING_LEVEL: 'DEBUG'
```
Comment (jacopocarlini) [inline docker/config/.env:2]:
```suggestion
APP_LOGGING_LEVEL=DEBUG
```

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -1,77 +1,60 @@
 version: '3.8'
 
-# TODO edit this file properly
 services:
   app:
     container_name: 'app'
-    image: ${image}
+    image: 'alessioselfcareint'
```
Comment (jacopocarlini) [inline docker/docker-compose.yml:6]:
```suggestion
    image: ${image}
```

File: src/main/resources/application-docker.properties [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Info
```
Comment (jacopocarlini) [inline src/main/resources/application-docker.properties:1]: delete this file and use `.env`

File: docker/docker-compose.yml (hunk 3 above)
Comment (alessio-acitelli) [reply docker/docker-compose.yml]: done

File: docker/config/.env (hunk 2 above)
Comment (alessio-acitelli) [reply docker/config/.env]: done

**Comment by alessio-acitelli** [comment]
update_code

**Review by jacopocarlini — APPROVED** [review]
Great work 🚀

**Pull request merged**